lib/gui/app.js

```diff
@@ -100,9 +101,12 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
     currentVersion: packageJSON.version
   });
 
-  UpdateNotifierService.isLatestVersion().then((isLatestVersion) => {
+  const CURRENT_VERSION = ManifestBindService.get('version');
 
-    if (isLatestVersion) {
+  s3Packages.getLatestVersion(s3Packages.BUCKET_URL_PRODUCTION, {
+    currentVersion: CURRENT_VERSION
+  }).then((latestVersion) => {
+    if (latestVersion === CURRENT_VERSION) {
```

Comment:
Or you could just return the "0.0.0" that it's already using as the "lowest possible semver value"?